slash_commands/misc.py

```diff
@@ -108,6 +108,41 @@ async def leetcode_leaderboard(ctx: discord.Interaction, order: str = "total"):
 
     await ctx.response.send_message(embed=embed, ephemeral=bot_context.ephemeral)
 
+@app_commands.describe(order="Sort leaderboard by different metrics")
+@app_commands.choices(order=[
+    app_commands.Choice(name="Total Commits this Year (default)", value="commits"),
+    app_commands.Choice(name="Total PRs this year", value="prs"),
```

Comment:
Can we add total comments too since I leave so many on every one of your prs?

Reply:
I'm not comfortable working with big data like that

Reply:
in all seriousness it would be difficult. getting number of comments on the user's PRs wouldn't be hard, but would probably get rate limited bc of pagination. haven't looked too deeply though